app/models/theme.rb

```diff
@@ -188,25 +201,28 @@ def self.clear_default!
     expire_site_cache!
   end
 
-  def self.transform_ids(ids, extend: true)
```

Comment:
Drop support for `extend` here. It increases the matrix for caching leading to extra requests.